src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -498,30 +497,83 @@ impl SimulationElement for A320HydraulicBrakingLogic {
     }
 }
 
+struct Door {
```

Comment:
It's good to note that both the `Door` and `PushbackTug` can likely be moved outside of "hydraulics" as they don't directly relate to it anymore. I might do that in another step of the refactoring, or leave that as an exercise to the reader 😉.

Reply:
Reader is happy :D